vagrant/app_setup.sh

```diff
@@ -8,9 +8,12 @@
 sudo -n dnf install -y pipenv
 
 cd /vagrant/vagrant
+export PIPENV_PIPFILE=/vagrant/vagrant/Pipfile
+mv /vagrant/requirements{.txt,}
```

Comment:
Could you please explain in the commit message or here in a comment, why it is needed?

Reply:
Thanks for the clarification.
I think we can work on this as a follow up, no need to block on it.